man/man1/p11sak.1.in

```diff
@@ -683,6 +684,11 @@ option. When option
 .BR \-\-long | \-l
 is specified, the keys are listed in long format, displaying the values of all
 attributes defined for the key type, including non-boolean attributes.
+When option
+.BR \-\-hsm\-mkvp | \-m
+is specified, the HSM master key verification patterns (MKVPs) of the keys are
+displayed. This is only possible for secure key tokens, such as the CCA and EP11
+tokens.
```

Comment:
We could add a hint that this is also only possible/applicable for private keys. Same below ... 
Displaying public keys with --long and --hsm-mkvp just silently ignores the hsm parm.

Reply:
Ah yes, and we should add a hint that the --hsm-mkvp option is only applicable together with --long

Reply:
No, both is not true.  

In EP11 a public key is a MACed SPKI which is also bound to the EP11 master key. For CCA, public keys are not bound to the master keys though.

And no, `--hsm-mkvp` is NOT only available with --long, it works also without `--long`. Actually I think the way more useful use case is without `--long`.

Reply:
OK, yes, I see it now, forget the two comments.